{
 "about": {
  "channels": [
   "https://conda.anaconda.org/conda-forge/osx-64",
   "https://conda.anaconda.org/conda-forge/noarch",
   "https://repo.continuum.io/pkgs/free/osx-64",
   "https://repo.continuum.io/pkgs/free/noarch",
   "https://repo.continuum.io/pkgs/pro/osx-64",
   "https://repo.continuum.io/pkgs/pro/noarch"
  ],
  "conda_build_version": "2.1.10",
  "conda_env_version": "4.2.13",
  "conda_private": false,
  "conda_version": "4.2.13",
  "dev_url": "https://git.openstack.org/cgit/openstack/keystoneauth",
  "doc_url": "https://docs.openstack.org/developer/keystoneauth/",
  "env_vars": {
   "CIO_TEST": "<not set>",
   "CONDA_DEFAULT_ENV": "root",
   "CONDA_ENVS_PATH": "<not set>",
   "DYLD_LIBRARY_PATH": "<not set>",
   "PATH": "/Users/travis/miniconda3/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p643/bin:/Users/travis/.rvm/gems/ruby-2.0.0-p643@global/bin:/Users/travis/.rvm/rubies/ruby-2.0.0-p643/bin:/Users/travis/.rvm/bin:/Users/travis/bin:/Users/travis/.local/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin",
   "PYTHONHOME": "<not set>",
   "PYTHONPATH": "<not set>"
  },
  "home": "https://docs.openstack.org/developer/keystoneauth/",
  "license": "Apache 2.0",
  "license_family": "Apache",
  "offline": false,
  "root_pkgs": [
   "conda-forge::anaconda-client-1.6.3-py36_0",
   "asn1crypto-0.22.0-py36_0",
   "conda-forge::beautifulsoup4-4.5.3-py36_0",
   "cffi-1.10.0-py36_0",
   "conda-forge::chardet-3.0.2-py36_1",
   "conda-forge::clyent-1.2.1-py36_0",
   "conda-forge::conda-4.2.13-py36_0",
   "conda-forge::conda-build-2.1.10-py36_0",
   "conda-forge::conda-env-2.6.0-0",
   "conda-forge::conda-forge-build-setup-4.4.4-0",
   "conda-forge::conda-verify-2.0.0-py36_0",
   "cryptography-1.8.1-py36_0",
   "conda-forge::filelock-2.0.6-py36_0",
   "idna-2.5-py36_0",
   "conda-forge::jinja2-2.9.5-py36_0",
   "conda-forge::markupsafe-1.0-py36_0",
   "openssl-1.0.2l-0",
   "packaging-16.8-py36_0",
   "pip-9.0.1-py36_1",
   "conda-forge::pkginfo-1.2.1-py36_0",
   "pycosat-0.6.2-py36_0",
   "pycparser-2.17-py36_0",
   "conda-forge::pycrypto-2.6.1-py36_1",
   "pyopenssl-17.0.0-py36_0",
   "pyparsing-2.1.4-py36_0",
   "python-3.6.1-2",
   "conda-forge::python-dateutil-2.6.0-py36_0",
   "conda-forge::pytz-2017.2-py36_0",
   "conda-forge::pyyaml-3.12-py36_1",
   "readline-6.2-2",
   "requests-2.14.2-py36_0",
   "ruamel_yaml-0.11.14-py36_1",
   "setuptools-27.2.0-py36_0",
   "six-1.10.0-py36_0",
   "sqlite-3.13.0-0",
   "tk-8.5.18-0",
   "wheel-0.29.0-py36_0",
   "xz-5.2.2-1",
   "yaml-0.1.6-0",
   "zlib-1.2.8-3"
  ],
  "summary": "Authentication Library for OpenStack Identity"
 },
 "conda_build_config": {},
 "files": [
  "lib/python3.5/site-packages/keystoneauth1/exceptions/auth.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/access.py",
  "lib/python3.5/site-packages/keystoneauth1/extras/_saml2/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/loading/utils.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/test_discovery.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/client_fixtures.py",
  "lib/python3.5/site-packages/keystoneauth1/exceptions/auth_plugins.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/loading/test_entry_points.py",
  "lib/python3.5/site-packages/keystoneauth1/fixture/v2.py",
  "lib/python3.5/site-packages/keystoneauth1/extras/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/saml2/utils.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/data/keystone_v3_sample_response.json",
  "lib/python3.5/site-packages/keystoneauth1/loading/session.py",
  "lib/python3.5/site-packages/keystoneauth1/session.py",
  "lib/python3.5/site-packages/keystoneauth1/extras/oauth1/_loading.py",
  "lib/python3.5/site-packages/keystoneauth1/extras/_saml2/v3/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/access/test_v2_service_catalog.py",
  "lib/python3.5/site-packages/keystoneauth1/access/service_catalog.py",
  "lib/python3.5/site-packages/keystoneauth1/hacking/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/saml2/fixtures/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/fixture/exception.py",
  "lib/python3.5/site-packages/keystoneauth1/loading/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/kerberos/test_fedkerb_loading.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/kerberos/test_mapped.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/kerberos/utils.py",
  "lib/python3.5/site-packages/keystoneauth1/loading/_utils.py",
  "lib/python3.5/site-packages/keystoneauth1/extras/kerberos/_loading.py",
  "lib/python3.5/site-packages/keystoneauth1/token_endpoint.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/test_betamax_serializer.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/exceptions/service_providers.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/loading/test_conf.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/data/README",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/loading/test_adapter.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/test_matchers.py",
  "lib/python3.5/site-packages/keystoneauth1/adapter.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/data/keystone_v2_sample_request.json",
  "lib/python3.5/site-packages/keystoneauth1/identity/v2.py",
  "lib/python3.5/site-packages/keystoneauth1/access/service_providers.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/v3/token.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/v3/tokenless_auth.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/test_token_endpoint.py",
  "lib/python3.5/site-packages/keystoneauth1/loading/opts.py",
  "lib/python3.5/site-packages/keystoneauth1/fixture/hooks.py",
  "lib/python3.5/site-packages/keystoneauth1/loading/base.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/generic/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/exceptions/connection.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/identity/utils.py",
  "lib/python3.5/site-packages/keystoneauth1/loading/_plugins/identity/generic.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/test_session.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/saml2/test_auth_saml2.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/kerberos/base.py",
  "lib/python3.5/site-packages/keystoneauth1/_utils.py",
  "lib/python3.5/site-packages/keystoneauth1/fixture/discovery.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/test_betamax_fixture.py",
  "lib/python3.5/site-packages/keystoneauth1/extras/oauth1/v3.py",
  "lib/python3.5/site-packages/keystoneauth1/exceptions/oidc.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/kerberos/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/identity/test_identity_common.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/identity/test_identity_v3_oidc.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/loading/test_v3.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/oauth1/test_oauth1.py",
  "lib/python3.5/site-packages/keystoneauth1/access/access.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/data/test_pre_record_hook.json",
  "lib/python3.5/site-packages/keystoneauth1/service_token.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/test_fixtures.py",
  "lib/python3.5/site-packages/keystoneauth1/extras/_saml2/v3/saml2.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/generic/base.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/loading/test_cli.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/test_service_token.py",
  "lib/python3.5/site-packages/keystoneauth1/exceptions/catalog.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/base.py",
  "lib/python3.5/site-packages/keystoneauth1/loading/conf.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/oauth1/test_oauth1_loading.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/saml2/test_auth_adfs.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/generic/password.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/v3/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/saml2/fixtures/templates/soap_response.xml",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/data/keystone_v2_sample_response.json",
  "lib/python3.5/site-packages/keystoneauth1/loading/adapter.py",
  "lib/python3.5/site-packages/keystoneauth1/loading/_plugins/identity/v3.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/identity/test_identity_v3_federation.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/identity/test_identity_v3.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/saml2/fixtures/templates/authn_request.xml",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/k2k_fixtures.py",
  "lib/python3.5/site-packages/keystoneauth1/fixture/v3.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/generic/token.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/loading/test_loading.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/utils.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/test_betamax_hooks.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/identity/test_token.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/data/ksa_serializer_data.json",
  "lib/python3.5/site-packages/keystoneauth1/loading/identity.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/access/test_v3_access.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/oidc_fixtures.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/loading/test_session.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/exceptions/test_exceptions.py",
  "lib/python3.5/site-packages/keystoneauth1/exceptions/http.py",
  "lib/python3.5/site-packages/keystoneauth1/exceptions/response.py",
  "lib/python3.5/site-packages/keystoneauth1/extras/oauth1/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/saml2/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/discover.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/saml2/fixtures/templates/saml_assertion.xml",
  "lib/python3.5/site-packages/keystoneauth1/exceptions/base.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/data/keystone_v3_sample_request.json",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/data/ksa_betamax_test_cassette.yaml",
  "lib/python3.5/site-packages/keystoneauth1/exceptions/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/v3/password.py",
  "lib/python3.5/site-packages/keystoneauth1/extras/_saml2/v3/base.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/saml2/examples/xml/ADFS_fault.xml",
  "lib/python3.5/site-packages/keystoneauth1/fixture/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1-2.21.0-py3.5.egg-info/pbr.json",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/identity/test_access.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/identity/test_identity_v2.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/oauth1/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/matchers.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/v3/federation.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/v3/k2k.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/identity/test_tokenless_auth.py",
  "lib/python3.5/site-packages/keystoneauth1/loading/_plugins/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/identity/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/identity/test_password.py",
  "lib/python3.5/site-packages/keystoneauth1/plugin.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/access/test_v2_access.py",
  "lib/python3.5/site-packages/keystoneauth1/hacking/checks.py",
  "lib/python3.5/site-packages/keystoneauth1/loading/_plugins/identity/v2.py",
  "lib/python3.5/site-packages/keystoneauth1/loading/_plugins/identity/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/v3/base.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/loading/test_generic.py",
  "lib/python3.5/site-packages/keystoneauth1/access/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1-2.21.0-py3.5.egg-info/not-zip-safe",
  "lib/python3.5/site-packages/keystoneauth1/exceptions/discovery.py",
  "lib/python3.5/site-packages/keystoneauth1/fixture/serializer.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/kerberos/test_v3.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/saml2/examples/xml/ADFS_RequestSecurityTokenResponse.xml",
  "lib/python3.5/site-packages/keystoneauth1/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/access/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/v3/totp.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/extras/_saml2/v3/adfs.py",
  "lib/python3.5/site-packages/keystoneauth1/fixture/keystoneauth_betamax.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/loading/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/test_utils.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/exceptions/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1-2.21.0-py3.5.egg-info/PKG-INFO",
  "lib/python3.5/site-packages/keystoneauth1/extras/_saml2/_loading.py",
  "lib/python3.5/site-packages/keystoneauth1/extras/kerberos/__init__.py",
  "lib/python3.5/site-packages/keystoneauth1/loading/_plugins/admin_token.py",
  "lib/python3.5/site-packages/keystoneauth1/loading/cli.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/keystoneauth_fixtures.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/test_hacking_checks.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/access/test_v3_service_catalog.py",
  "lib/python3.5/site-packages/keystoneauth1/tests/unit/extras/kerberos/test_kerberos_loading.py",
  "lib/python3.5/site-packages/keystoneauth1/identity/v3/oidc.py"
 ],
 "index": {
  "arch": "x86_64",
  "build": "py35_0",
  "build_number": 0,
  "depends": [
   "iso8601 >=0.1.11",
   "pbr !=2.1.0,>=2.0.0",
   "positional >=1.1.1",
   "python 3.5*",
   "requests !=2.12.2,!=2.13.0,>=2.10.0",
   "six >=1.9.0",
   "stevedore >=1.20.0"
  ],
  "license": "Apache 2.0",
  "license_family": "Apache",
  "name": "keystoneauth1",
  "platform": "osx",
  "subdir": "osx-64",
  "version": "2.21.0"
 },
 "metadata_version": 1,
 "name": "keystoneauth1",
 "raw_recipe": "{% set name = \"keystoneauth1\" %}\n{% set version = \"2.21.0\" %}\n{% set bundle = \"tar.gz\" %}\n{% set hash_type = \"sha256\" %}\n{% set hash = \"1955971f51e713a89e0789a1ff3d30a3a717a7af4ad4e97b36cefb9046f4f5f8\" %}\n{% set build = 0 %}\n\npackage:\n  name: {{ name }}\n  version: {{ version }}\n\nsource:\n  fn: {{ name }}-{{ version }}.{{ bundle }}\n  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}\n  {{ hash_type }}: {{ hash }}\n\nbuild:\n  number: {{ build }}\n  script: python setup.py install --single-version-externally-managed --record=record.txt\n\nrequirements:\n  build:\n    - python\n    - setuptools\n    - pbr !=2.1.0,>=2.0.0\n\n  run:\n    - python\n    - pbr !=2.1.0,>=2.0.0\n    - iso8601 >=0.1.11\n    - positional >=1.1.1\n    - requests !=2.12.2,!=2.13.0,>=2.10.0\n    - six >=1.9.0\n    - stevedore >=1.20.0\n    \ntest:\n  imports:\n    - keystoneauth1\n    - keystoneauth1.identity\n    - keystoneauth1.identity.v2\n    - keystoneauth1.identity.v3\n    - keystoneauth1.identity.generic\n    - keystoneauth1.extras\n\nabout:\n  home: https://docs.openstack.org/developer/keystoneauth/\n  license_file: LICENSE\n  license: Apache 2.0\n  license_family: Apache\n  summary: Authentication Library for OpenStack Identity\n  dev_url: https://git.openstack.org/cgit/openstack/keystoneauth\n  doc_url: https://docs.openstack.org/developer/keystoneauth/\n\nextra:\n  recipe-maintainers:\n    - pmlandwehr\n",
 "rendered_recipe": {
  "about": {
   "dev_url": "https://git.openstack.org/cgit/openstack/keystoneauth",
   "doc_url": "https://docs.openstack.org/developer/keystoneauth/",
   "home": "https://docs.openstack.org/developer/keystoneauth/",
   "license": "Apache 2.0",
   "license_family": "Apache",
   "license_file": "LICENSE",
   "summary": "Authentication Library for OpenStack Identity"
  },
  "build": {
   "noarch": "",
   "noarch_python": false,
   "number": "0",
   "script": "python setup.py install --single-version-externally-managed --record=record.txt",
   "string": "py35_0"
  },
  "extra": {
   "final": true,
   "recipe-maintainers": [
    "pmlandwehr"
   ]
  },
  "package": {
   "name": "keystoneauth1",
   "version": "2.21.0"
  },
  "requirements": {
   "build": [
    "ca-certificates 2017.4.17 0",
    "certifi 2017.4.17 py35_0",
    "ncurses 5.9 10",
    "openssl 1.0.2k 0",
    "pbr 3.0.1 py35_0",
    "pip 9.0.1 py35_0",
    "python 3.5.3 3",
    "readline 6.2 0",
    "setuptools 33.1.1 py35_0",
    "sqlite 3.13.0 1",
    "tk 8.5.19 1",
    "wheel 0.29.0 py35_0",
    "xz 5.2.2 0",
    "zlib 1.2.11 0"
   ],
   "run": [
    "python",
    "pbr !=2.1.0,>=2.0.0",
    "iso8601 >=0.1.11",
    "positional >=1.1.1",
    "requests !=2.12.2,!=2.13.0,>=2.10.0",
    "six >=1.9.0",
    "stevedore >=1.20.0"
   ]
  },
  "source": {
   "fn": "keystoneauth1-2.21.0.tar.gz",
   "sha256": "1955971f51e713a89e0789a1ff3d30a3a717a7af4ad4e97b36cefb9046f4f5f8",
   "url": "https://pypi.io/packages/source/k/keystoneauth1/keystoneauth1-2.21.0.tar.gz"
  },
  "test": {
   "imports": [
    "keystoneauth1",
    "keystoneauth1.identity",
    "keystoneauth1.identity.v2",
    "keystoneauth1.identity.v3",
    "keystoneauth1.identity.generic",
    "keystoneauth1.extras"
   ]
  }
 },
 "version": "2.21.0"
}